content/bn/docs/reference/glossary/kube-scheduler.md

```diff
@@ -0,0 +1,23 @@
+---
+title: কুব-সিডিউলার​
+id: kube-scheduler
+date: ২০১৮-০৪-১২
+full_link: /bn/docs/reference/command-line-tools-reference/kube-scheduler/
```

Comment:
Does this link exist in [kubernetes:dev-1.24-bn.1](https://github.com/kubernetes/website/tree/dev-1.24-bn.1)?
If not, we should hold this PR until that link works.

@asem-hamid

Reply:
Actually we don't have to hold , we can simply omit `/bn ` from link  .  Full page  is not mandatory for glossary  terms .

Reply:
Ok, That will be great.